surveys/2021-annual-survey/questions.md

```diff
@@ -475,49 +480,82 @@ Challenges:
-- I've read TWiR
-- I've read GH rust-lang org/Zulip/Discord/internals/users/r/rust (separate answers or one answer?)
-- I've written a comment/post on GH rust-lang org/Zulip/Discord/internals/users/r/rust (separate answers or one answer?)
-- Attended a meetup (including virtual)
-- participated in a private Rust community (e.g., work Slack, private messages)
+- I have produced informational content about Rust (e.g., blogged, live streamed, made a YouTube video, etc.).
+- On several occasions I have consumed informational content about Rust (e.g., blogs, live streams, YouTube videos, etc.).
+- On several occasions I have read *comments* about Rust content on "news" sites (e.g., Hacker News, reddit.com/r/rust, lobste.rs/t/rust, etc.)
+- On several occasions I have *commented* on about Rust content on "news" sites (e.g., Hacker News, reddit.com/r/rust, lobste.rs/t/rust, etc.)
+- On several occasions I have read official Rust communication channels (e.g., This Week in Rust, the official Rust blog, the Rust Twitter account, etc.)
+- On several occasions I have participated in conversations about Rust on social media (Twitter, Facebook, LinkedIn, etc.)
+- I have participated in Rust community forums or chats (e.g., users.rust-lang.org, Rust Discord, a local Rust chat community, etc.)
+- I have attended a Rust meetup or conference (virtual or in-person).
 
 > **justification**
-> We'd like to get a picture of _how_ people participate in the Rust community
 >
-> TODO separate issue management, docs, etc. from code contribution?
-> 
-> TODO possibly multiple questions for past month, past year, ever, + would you like to do
-> POSSIBLE Approach below:
+> We'd like to get a picture of _how_ people participate in the Rust community. In
+> particular we can use this information to do cohort analysis on highly "active"
+> community members in comparison to less active community members.
 
-> **Survey Flow Example** Following question would only appear if indicated "I have read the Rust blog."
+### How often do you actively contribute to the Rust project?
 
-### Previously you indicated that you read the Rust blog. How often do you do so?
+Options:
+
+- Comment on, contribute to discussion of, or provide edits to an open RFC
+- Create a new thread or comment on internals.rust-lang.org
+- Discuss Rust project  in an official chat (either Zulip or Discord)
+- Open an issue on any repo in the rust-lang GitHub org
+- Contribute code changes (including tests) to the Rust compiler (rust-lang/rust)
+- Contribute code changes (including tests) to any other project in the rust-lang GitHub org
+- Contribute non-code changes (documentation, comments, etc.) to any project in the rust-lang GitHub org.
 
-- [frequency options]
+Select one:
 
-> **Survey Flow Example** Following question would only appear if indicated "I have not read the Rust blog."
+- Almost daily
+- Weekly
+- Around once a month
+- Every other month or so
+- Never or rarely but have tried
```

Comment:
I would change "Every other month or so" to something like "less frequently than monthly", and this option to "Never, but have tried" and likewise for the next